examples/src/main/scala/com/example/http4s/ExampleService.scala

```diff
@@ -161,20 +159,6 @@ class ExampleService[F[_]](implicit F: Async[F]) extends Http4sDsl[F] {
           Ok(s"Form Encoded Data\n$s")
         }
 
-      // /////////////////////////////////////////////////////////////
-      // ////////////////////// Server Push //////////////////////////
-      case req @ GET -> Root / "push" =>
-        // http4s intends to be a forward looking library made with http2.0 in mind
```

Comment:
lol

Reply:
I find this graceful tbh https://github.com/http4s/http4s/pull/6247/files#diff-1dbdae8fc5bc8c6050d06f5ab398d31356251cae7406c5ac3bb177d4d0e02715L165-L178